lib/cli/src/frameworks/svelte/Button.stories.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+  import { Meta, Template, Story } from "@storybook/addon-svelte-csf";
+  import Button from "./Button.svelte";
+</script>
+
+<Meta
+  title="Example/Button"
+  component={Button}
+  argTypes={{
+    label: { control: "text" },
+    primary: { control: "boolean" },
+    backgroundColor: { control: "color" },
+    size: {
+      control: { type: "select", options: ["small", "medium", "large"] },
+    },
+    onClick: { action: "onClick" },
+  }}
+/>
+
+<Template let:args>
+  <Button {...args} on:click={args.onClick} />
```

Comment:
I have seen this in a lot of examples in Storybook, but what is the purpose of declaring events as properties ? It's look like a React convention.

In Svelte, event and properties are not the same thing. It's not a big deal, but passing an undeclared property to a component in svelte will display a warning in development mode.

Reply:
Args can be properties, events, slots, etc.

The two main patterns for how we use them, tho we may develop more:
1) auto-generated controls to manipulate the value in UI
2) dependency injection, e.g. auto-generated actions, which makes the stories more portable (no direct dependency on `@storybook/addon-actions`

It's the latter case that's highlighted here

Reply:
Thanks for the clarification. 

Something should probably be added to the plugin in order to remove this warning. maybe filtering actions, and subscribing automatically to the linked event.